packages/shared/lib/core/profile-manager/api/getVotingEventIds.ts

```diff
@@ -0,0 +1,10 @@
+import { get } from 'svelte/store'
+
+import { EventId, ParticipationEventType } from '@iota/wallet/out/types'
+
+import { profileManager } from '../stores'
+
+export function getVotingEventIds(): Promise<EventId[]> {
+    const manager = get(profileManager)
+    return manager?.getParticipationEventIds(ParticipationEventType.Voting)
+}
```

Comment:
This is dead code. We shouldn't create wrappers around wallet.rs API, just for the sake of having them. We might never need it, so why go through the effort?

Reply:
I agree, although I plan on using it for #4862, so it should be fine. I admit I did get a bit ahead of myself since I knew I'd use it soon :sweat_smile: